app/views/news_tematica/news_tematicas/new.html.haml

```diff
@@ -3,8 +3,9 @@
 = form_for @news_tematica do |f|
   = render 'web/muestra_errores_form', variable: @news_tematica
   %table
-    - tematicas = NewsTematica::NewsTematicaDecorator.tematicas_dropdown
-    = campo f, 'tematica_id', tipo: 'dropdownoptions', datos: tematicas, prompt: 'Selecciona una'
+    - suscribibles = Suscribir::Newsletter.all + Tematica::Tematica.all
+    = f.collection_select :tematica_id, suscribibles, :id_y_clase, :nombre, { selected: params[:tematica_id] }
```

Comment:
I don't think it can work; you'll be saving the same value, 1, for both the NewsGeneral and the first Tematica... I guess that the way to do things right is changing `tematica_id` to `suscribible_id + suscribible_type`